phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/app/PaymentsManager.kt

```diff
@@ -29,8 +30,29 @@ class PaymentsManager(
 ) : CoroutineScope by MainScope() {
     private val log = newLogger(loggerFactory)
 
-    /** A flow containing a list of payments, directly taken from the database and automatically refreshed when the database changes. */
-    internal val payments = MutableStateFlow<List<WalletPayment>>(emptyList())
+    data class PaymentsPage(
```

Comment:
A comment in `HomeView.swift` mentions that incrementing the offset is not yet possible due to a limitation in SwiftUI. So in fact, for now, offset is always 0. Maybe we should add a comment here too?

Reply:
Done in f170706